relayer/src/ethereum_checkpoints/mod.rs

```diff
@@ -186,10 +168,15 @@ pub async fn relay(args: RelayCheckpointsArgs) {
                 }
             }
             Ok(Err(sync_update::Error::ReplayBackRequired { .. })) => {
-                log::info!("Replay back within the main loop. Exiting");
+                log::error!("Replay back within the main loop. Exiting");
                 return;
             }
-            Ok(Err(e)) => log::info!("The program failed with: {e:?}. Skipping"),
+            Ok(Err(e)) => {
+                log::info!("The program failed with: {e:?}. Skipping");
+                if let sync_update::Error::NotActual = e {
+                    slot_last = slot;
+                }
```

Comment:
Why not a separate case in `match`?

Reply:
not to duplicate log record

Reply:
But the case when this error appears is not a error, as I see, more like expected behaviour

Reply:
yep, just like others from this arm

Reply:
Why? When the error don't match specific variant it seems like unexpected behaviour